tests/Console/Command/DescribeCommandTest.php

```diff
@@ -32,18 +34,31 @@
  */
 final class DescribeCommandTest extends TestCase
 {
+    /**
+     * @var Application
+     */
+    private $application;
+
+    protected function setUp()
+    {
+        $this->application = new Application();
```

Comment:
> its a private property, side effects can only be occur on purpose

By side effect I mean: the instance is used in some way in a test that may affect another test as they share the same instance.

> PHPUnit should release the test class itself, clearing its properties as well. There is no other way of releasing the memory (i.e. setting it to null does not help)

Not sure it's PHPUnit's responsability to do that, could make sense. But I'm pretty sure it doesn't currently. Why setting the property to `null` wouldn't help?